pom.xml

```diff
@@ -257,6 +257,22 @@
         <enabled>false</enabled>
       </snapshots>
     </repository>
+    <repository>
```

Comment:
Yeah I get it. At this point though, would it just be easier to declare the Google repo above `central` above, and not use the mirror settings?

Reply:
Ur, I'm not sure that we can declare `Google Maven Central` as Apache Spark default repo because not everyone welcome `Google` repo. In Apache Spark, we only use this in GitHub Action. The other build environment, we don't use `Google Maven Central` at all.

Reply:
Are you sure, @srowen ?

Reply:
OK, it could be listed second. I don't see a problem with including a mirror as a repository here, as it's just a mirror of central. (We have had in the past, even, vendor-specific repos.) I hope it would have the same effect in improving robustness as anyone running the build would have it look at both repos before failing.